src/main/webapp/creatives/maps/simid_map.css

```diff
@@ -43,7 +43,7 @@ html, body {
     width: 80%;
     margin: auto;
     position: relative;
-    top: 50%;
+    top: 215;
```

Comment:
I just sent out a PR for styling, when that gets approved we should see if this is the proper top for the map